vxquery-core/src/main/java/org/apache/vxquery/runtime/functions/cast/AbstractCastToOperation.java

```diff
@@ -226,4 +226,8 @@ public void convertNormalizedString(UTF8StringPointable stringp, DataOutput dOut
     public void convertToken(UTF8StringPointable stringp, DataOutput dOut) throws SystemException, IOException {
         throw new SystemException(ErrorCode.XPTY0004);
     }
+
+    public void convertNull(DataOutput dOut) throws SystemException, IOException {
```

Comment:
I think I have already added a testcase for null with fn:data in file fn_data1.xq. What do you mean by "this options"? Does anything change if I add null to CastableAsStringOperation that needs to be tested?